### ISSUE TITLE
x/pkgsite: multiple invalid HTML

### ISSUE THREAD

**Issue by 3052** [golang/go#70730]
### What is the URL of the page with the issue?

~~~
view-source:https://pkg.go.dev/41.neocities.org/text
~~~

### What is your user agent?

Firefox

### Screenshot

> ![invalid](https://github.com/user-attachments/assets/9388bece-958a-4034-add2-dcf65de5624c)

### What did you see happen?

first is the double close tag:

https://github.com/golang/pkgsite/blob/dc08416c8122836b0f06f5b3025699f37e3d53b9/static/shared/header/header.tmpl#L27-L50

then space is removed on both sides, leading to attributes with no separating space:

<https://github.com/golang/pkgsite/blob/dc08416c8122836b0f06f5b3025699f37e3d53b9/static/frontend/unit/main/_meta.tmpl#L61-L69>

and another example actually hard coded:

https://github.com/golang/pkgsite/blob/dc08416c8122836b0f06f5b3025699f37e3d53b9/static/shared/footer/footer.tmpl#L142

### What did you expect to see?

valid HTML

**Comment by 3052** [comment]
https://github.com/golang/pkgsite/commit/e388c1678c2f85e12c2e88a05c71ab90039edf50

**Comment by findleyr** [comment]
Thank you for your diligence. It looks like you have already done the hard work of finding and root causing these problems. Would you like to fix them? We would welcome a contribution fixing these errors.

**Comment by 3052** [comment]
I don't see that you're on the Go team...

**Comment by findleyr** [comment]
@3052 I'm not sure what you mean. 

https://github.com/orgs/golang/teams/google-go-team

**Comment by 3052** [comment]
![Untitled](https://github.com/user-attachments/assets/00568e7b-8d3c-4137-9fe8-7633a0b2b381)

**Comment by ianlancetaylor** [comment]
When I look at https://github.com/findleyr I see two organizations listed: Go and Googlers.  @3052 Do you not see that?  I wonder if there is some GitHub setting that hides it in some cases.

![Screenshot from 2024-12-12 12-52-07](https://github.com/user-attachments/assets/e1af7d6c-fe5b-4a56-b04b-1f5f7c0c9a42)

**Comment by jimmyfrasche** [comment]
I only see Go listed and if I repopen the profile in a private window I see neither.

**Comment by ianlancetaylor** [comment]
Huh, if I use a Chrome incognito window I don't see any organizations for @findleyr, and for myself I see only the SWIG organization.  I just edited my visibility for the golang organization, which I did from the organization page, and now I see myself listed in that organization.  So it seems to be up to each person, or each organization, whether to publicize their organization membership.

**Comment by 3052** [comment]
WOW that is really annoying - I am a member of two as well - but both were private by default - so I just fixed myself

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership

**Comment by findleyr** [comment]
Thanks, I did not know that! I've made my Go membership public now.

**Comment by FuenRob** [comment]
Hello, can I make a contribution to solve this issue?

**Comment by Mahimachavalam009** [comment]
can i contribute?

**Comment by findleyr** [comment]
Hi @FuenRob and @Mahimachavalam009 we would welcome fixes for these issues.

I think one CL will do, so @FuenRob since you asked first could you please confirm you will work on it? If not, no worries; @Mahimachavalam009 can contribute in that case.

**Comment by FuenRob** [comment]
Yes @findleyr I want to contribute by solving this fix. Thanks. 

Should I assign the issue?

**Comment by findleyr** [comment]
@FuenRob I already assigned it to you :)

**Comment by FuenRob** [comment]
Hi @findleyr 

I've uploaded the changes via Gerrit. How can I link the change to this issue?

Here is the link to the change: https://go-review.googlesource.com/c/pkgsite/+/639075

Please, any problem or failure, do not hesitate to tell me to fix it.

**Comment by ianlancetaylor** [comment]
@FuenRob In repo's other than the main go repo, you should write

    Fixes golang/go#70730

See https://go.dev/wiki/CommitMessage.

**Comment by FuenRob** [comment]
Hello, do I need to do anything else?

**Comment by jba** [comment]
The bad span tags are obviously a problem.

What is the concern with the hardcoded href?

**Comment by FuenRob** [comment]
Hi @jba , yes the hardcoded URLs are already resolved and are with environment variables in the internal/config/serverconfig/config.go file
